src/scenes/VapaaSivistystyo/components/Koulutusala.js

```diff
@@ -0,0 +1,102 @@
+import _ from 'lodash'
+import React, { Component } from 'react'
+import styled from 'styled-components'
+
+import arrow from '../../../static/images/koulutusala-arrow.svg'
+import { COLORS } from "../../../modules/styles"
+import Tutkinto from './Tutkinto'
+
+const Wrapper = styled.div`
+  margin: 4px 0;
+  background-color: ${COLORS.BG_GRAY};
+  max-width: 625px;
```

Comment:
Oh no! Pixel values here.

Reply:
Koulutusala will be removed and this will be thus fixed.